CorsixTH/Lua/strings.lua

```diff
@@ -331,10 +331,11 @@ function Strings:setupAdviserMessage(messages)
     --build_advice
     --cheats
   }
+  -- Passes adviser message through string.format process
   local formatFunc
-  formatFunc = function(format_self, arg)
+  formatFunc = function(format_self, ...)
```

Comment:
Did you try changing the language with this change?

Afaik, the language strings are not plain text, they remember how they were constructed so after a language change they cab rebuild the displayed text by doing the same construction process again.

That construction process may assume there is one parameter for `format`, and this change would then break changing a language.

Reply:
Also, I'd merge 335 and 336, and have an empty line above 334  :)

(I am fine with the current  solution too, it's just  a suggestion.)